index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-unused-vars */
+const form = document.querySelector('#form');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBtn = document.querySelector('#addBtn');
+const bookContainer = document.getElementById('displayBook');
+let allBooks;
+
+const displayBooks = () => {
+  if (localStorage.getItem('allBooks') == null) {
+    allBooks = [];
+  } else {
+    allBooks = JSON.parse(localStorage.getItem('allBooks'));
+  }
+
+  allBooks.forEach((book) => {
+    bookContainer.innerHTML += `
+        <div>
+        <p>${book.title}</p>
+        <p>${book.author}</p>
+        <button onclick="remove(${book.id})">Remove</button>
+        <hr />
+        </div>`;
+  });
+};
+
+const add = (Title, Author) => {
+  const idRandom = Math.floor(Math.random() * 100000);
+  allBooks.push({ id: idRandom, title: Title, author: Author });
+  localStorage.setItem('allBooks', JSON.stringify(allBooks));
+  displayBooks();
+};
+
+window.addEventListener('DOMContentLoaded', () => {
+  displayBooks();
+});
+
+form.addEventListener('submit', () => {
+  const titleValue = title.value;
+  const authorValue = author.value;
+  add(titleValue, authorValue);
+});
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to implement book addition and removal without reloading the page everytime.